### PR TITLE
ticketbuyer: float64 -> dcrutil.Amount conversions

### PR DESCRIPTION
**Pull request by dajohi**

### CODE REVIEW

File: rpc/rpcserver/server.go [hunk 1]
```diff
@@ -1657,10 +1657,16 @@ func (t *ticketbuyerServer) SetMaxPriceRelative(ctx context.Context, req *pb.Set
 	if err != nil {
 		return nil, err
 	}
-	if req.MaxPriceRelative < 0 {
+	maxPriceRelative, err := dcrutil.NewAmount(req.MaxPriceRelative)
```
Comment (alexlyp) [inline rpc/rpcserver/server.go]: maxPriceRelative is NOT a dcrutil.Amount, it's a perecentage so it must be left as a float64

File: ticketbuyer/buyer.go [hunk 2]
```diff
@@ -505,49 +510,48 @@ func (t *TicketPurchaser) Purchase(height int64) (*PurchaseStats, error) {
...
+		return ps, err
+	}
+
 	log.Tracef("Estimated stake diff: (min: %v, expected: %v, max: %v)",
 		sDiffEsts.Min, sDiffEsts.Expected, sDiffEsts.Max)
 
 	// Set the max price to the configuration parameter that is lower
 	// Absolute or relative max price
 	var maxPriceAmt dcrutil.Amount
-	maxPriceAbs := t.MaxPriceAbsolute().ToCoin()
```
Comment (alexlyp) [inline ticketbuyer/buyer.go]: This stuff should probably stay the same

Comment (dajohi) [inline ticketbuyer/buyer.go]: I think it is good.

**Comment by dajohi** [comment]
Updated with maxPriceRelative reverted to float64

**Pull request merged**